assets/src/dashboard/app/views/shared/bodyViewOptions.js

```diff
@@ -48,28 +49,19 @@ const StorySortDropdownContainer = styled.div`
   align-self: flex-end;
 `;
 
-const SortDropdown = styled(Dropdown)`
-  min-width: 210px;
-`;
-
 const ControlsContainer = styled.div`
   display: flex;
   justify-content: center;
   align-items: center;
 `;
 
-const Label = styled.span`
-  ${TypographyPresets.Small};
-  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.gray500};
+const StyledLink = styled(Link)`
+  font-weight: 700;
```

Comment:
I just updated the typography presets to match figma again in this pr: https://github.com/google/web-stories-wp/pull/6484, can you remove this font-weight and we'll just let that happen once the type update's merged.